index.html

```diff
@@ -4240,47 +4240,54 @@ <h2>HTTP</h2>
         </section>
 
         <section class="informative">
-          <h2>JSON Processing</h2>
-
+          <h2>JSON-LD Processing</h2>
           <p>
 While the media types describing conforming documents defined in this
-specification always express JSON-LD, JSON-LD processing is not required to be
-performed, since JSON-LD is JSON. Some scenarios where processing a
-<a>verifiable credential</a> or a <a>verifiable presentation</a> as JSON is
+specification always express JSON-LD, RDF processing is not required to be
+performed, since JSON-LD is a concrete RDF syntax as described in [RDF11-CONCEPTS]. 
+Hence, a JSON-LD document is both an RDF document and a JSON document and correspondingly represents an instance of an RDF data model.
+See <a href="JSON-LD11##relationship-to-rdf">Relationship to RDF</a> for more details.
+          </p>
```

Comment:
```suggestion
          </p>
           <p>
RDF is a technology for modeling graphs of statements. Each statement is a single subject-property-value
relationship, which is referred to as a <a>claim</a> in this specification. JSON-LD is a technology that
enables the expression of RDF using idiomatic JSON, enabling developers familiar with JSON to write
applications to consume RDF as JSON. In general, subjects are expressed as JSON objects with each property and value of the subject as a JSON key and value, respectively. A special affordance is made to express an identifier of a subject, if present, using the <code>@id</code> JSON key, which is aliased in this specification to <code>id</code>.
See <a href="JSON-LD11##relationship-to-rdf">Relationship to RDF</a> for more details.
          </p>
           <p>
As JSON can be used to express all different kinds of information, a consumer of a particular JSON
document can only properly interpret the authors intent if they possess information that contextualizes
and disambiguates it from other possible expressions. Information to assist with this interpretation can either 
be wholly external to the JSON document or linked from within it. Compact JSON-LD documents include a <code>@context</code> property that internally expresses or links to contextual information and, per the JSON-LD specification, these documents follow object-oriented organizational principles to express <a>claims</a>. These features enable generalized processors to be written to convert JSON-LD documents from one context to another, but this is not needed when consumers receive JSON-LD documents that already use the context and shape that they expect. Authors of JSON-LD documents, such as <a>issuers</a> of <a>verifiable credentials</a>, are required to provide proper JSON-LD contexts and follow these rules in order to facilitate interoperability.
          </p>
           <p>
The text below helps consumers understand how to ensure a JSON-LD document is expressed in a context
and shape that their application already understands such that they do not need to transform it in
order to consume its contents. Notably, this does not mean that consumers do not need to understand any context at all, rather, consuming applications only need to understand a chosen set of contexts and document shapes to work with and not others. Issuers can publish contexts and information about their <a>verifiable credentials</a> to aid consumers who do not use generalized processors, just like would be done with any other JSON-formatted data.
          </p>
```

Reply:
Please add this suggestion as it provides more important background that makes clear what this section is about and that the consumer is always consuming RDF expressed in a JSON format when they consume JSON-LD -- and that contextual information must always be understood. This has been a consistent point of confusion and it's important to have this text.

Reply:
MInor:

```
<a href="JSON-LD11##relationship-to-rdf"
```

should be

```
<a href="JSON-LD11#relationship-to-rdf"
```

Reply:
@OR13, please respond to this.